editor/src/clj/editor/ui.clj

```diff
@@ -750,14 +753,43 @@
   (value [this] (.getValue this))
   (value! [this val] (with-on-edit-event-suppressed! this (.setValue this val))))
 
+(declare bind-key!)
+
 (extend-type TextField
   HasAction
-  (on-action! [node fn]
```

Comment:
it's better avoid using `fn` as a function argument, because it shadows `fn` keyword